cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -298,12 +296,45 @@ def test_resume_retired_instance(retire_vm, provider, remove_date):
     assert retire_vm.is_retired is False
 
 
+@pytest.mark.tier(2)
+@pytest.mark.long_running
+@test_requirements.multi_region
+@test_requirements.retirement
+def test_vm_retirement_from_global_region(setup_multi_region_cluster,
+                                          multi_region_cluster,
+                                          activate_global_appliance,
+                                          setup_remote_provider,
+                                          retire_vm):
+    """
+    retire a vm via CA
+
+    Polarion:
+        assignee: izapolsk
+        casecomponent: Provisioning
+        initialEstimate: 1/3h
+        testSteps:
+            1. Have a VM created in the provider in the Remote region
+               subscribed to Global.
+            2. Retire the VM using the Global appliance.
+        expectedResults:
+            1.
+            2. VM transitions to Retired state in the Global and Remote region.
+
+    """
+    retire_times = dict()
```

Comment:
Why not just?
```python
retire_times = {"start": generate_retirement_date_now() + timedelta(minutes=-5)}
```

Reply:
this is copypasta of retire_now test. It's just run against multi-region appliance